src/components/Netflix.js

```diff
@@ -1,16 +1,36 @@
-// import React from "react";
-// import Header from "./Header.js";
-// import Navbar from "./Navbar.js";
-// import "./Netflix.css";
+import React from "react";
+import "./Netflix.css";
+import Net from "../video/netflix.mp4";
+import Header from "./Header.js";
+import Navbar from "./Navbar.js";
 
-// function Netflix() {
-//   return (
-//     <div>
-//       <Header />
-//       <Navbar />
-//       <h1>NETFLIX</h1>
-//     </div>
-//   );
-// }
+function Netflix() {
+  return (
+    <div className="container">
+      <video
+        autoPlay
+        loop
+        muted
+        style={{
+          position: "absolute",
+          width: "100%",
+          height: "100%",
+          objectFit: "cover",
+          opactiy: "0.2",
+          zIndex: "-100",
+        }}
+      >
+        <source src={Net} type="video/mp4" />
+      </video>
```

Comment:
Could probably factor this component out into its own component and piss. the info in as props. seems like it is pretty similar everywhere

Reply:
video element inline styling removed, added to App.css. Hesitant to make it its own component.